test/e2e/quadlet_test.go

```diff
@@ -571,6 +571,7 @@ var _ = Describe("quadlet system generator", func() {
 		Entry("remap-auto2.container", "remap-auto2.container"),
 		Entry("remap-keep-id.container", "remap-keep-id.container"),
 		Entry("remap-keep-id2.container", "remap-keep-id2.container"),
+		Entry("workingdir.container", "workingdir.container"),
```

Comment:
this file is not commited

Reply:
and please flop at least volume/workingdir